### DISCUSSION TITLE
Why can't @babel/parser parse objects? Can someone help me?

### DISCUSSION THREAD

**Discussion by wood3n** [babel/babel#13937] [Q&A] [answered]
When i input an object in astexplorer like below code, this will get a mistake: Missing semicolon. (3:9), what is that mean? @babel/parser can't parse object correctly?
```js
{
  name: 'test',
  company: 'test company',
}
```
![image](https://user-images.githubusercontent.com/31716713/140640007-a48b0f76-5cbe-4b2d-9cbc-bcd98a0086dc.png)

**Comment by TrySound** [comment, marked as answer]
Because it's not an object. It's block. To get an object you should wrap curly brackets with parentheses.

**Reply by wood3n** [reply]
> To get an object you should wrap curly brackets with parentheses.

Thank you. like this? but it still has a problem.
![image](https://user-images.githubusercontent.com/31716713/140641350-c66298de-2608-4589-86e1-de3f880743e1.png)

**Reply by wood3n** [reply]
> Because it's not an object. It's block. To get an object you should wrap curly brackets with parentheses.

sorry, it's my fault, you are right, i reset this set's configuration, so that it had that problem.
